frontend/src/components/Login.js

```diff
@@ -10,8 +10,8 @@ import * as Lang from '../constants/langEnUs';
 import CallableModal from '../components/CallableModal';
 
 // Styles
-import 'bootstrap/dist/css/bootstrap.css';
-import 'styles/index.scss';
+// import 'bootstrap/dist/css/bootstrap.css';
```

Comment:
we should remove all references to bootstrap and just fix what breaks

Reply:
Removed the component all together as a new component was created.